tests/e2e/dsc_creation_test.go

```diff
@@ -424,6 +429,42 @@ func (tc *testContext) testDefaultCertsAvailable() error {
 	return nil
 }
 
+func (tc *testContext) testDefaultModelRegistryCertAvailable() error {
+	// Get expected cert secrets
+	defaultIngressCtrl, err := cluster.FindAvailableIngressController(tc.ctx, tc.customClient)
```

Comment:
We would need to add a check to verify if ModelRegistry component state is set to `Managed` before checking for secrets.

Reply:
Added check for managed state.